src/uniprotkb/components/protein-data-views/LigandDescriptionView.tsx

```diff
@@ -0,0 +1,86 @@
+import { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+
+import { RichText } from './FreeTextView';
+import ExternalLink from '../../../shared/components/ExternalLink';
+
+import { LocationToPath, Location } from '../../../app/config/urls';
+import externalUrls from '../../../shared/config/externalUrls';
+
+// For Ligand and LigandPart context refer to:
+// https://github.com/ebi-uniprot/uniprot-manual/blob/main/release-notes/2022-08-03-release.md#structuring-of-binding-site-annotations
+
+// A good example with well populated Ligand fields is Q9ZGI5
+export type Ligand = {
+  name: string;
+  id?: string;
+  label?: string;
+  note?: string;
+};
+
+export type LigandPart = {
+  name?: string;
+  id?: string;
+  label?: string;
+  note?: string;
+};
+
+type LigandViewProps = {
+  ligand: Ligand | LigandPart;
+};
+const LigandView = ({ ligand }: LigandViewProps) => {
+  const id = ligand.id?.replace('ChEBI:', '');
+  return (
+    <>
+      <RichText>{ligand.name}</RichText>
+      {ligand.label && ` ${ligand.label}`}
+      {id && (
+        <>
+          {' ('}
+          <Link
+            to={{
+              pathname: LocationToPath[Location.UniProtKBResults],
+              search: `query=ft_binding:"${id}"`,
+            }}
+          >
+            UniProtKB
+          </Link>
+          {' | '}
+          <ExternalLink url={externalUrls.ChEBI(id)}>ChEBI</ExternalLink>
+          {') '}
+        </>
+      )}
+      <RichText>{ligand.note}</RichText>
```

Comment:
Should the note here be preceded with a semicolon? as the note is similar to the description. Example: UniRule UR000177404